api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimService.java

```diff
@@ -50,41 +46,33 @@
             EligibilityStatus.INELIGIBLE, ClaimStatus.REJECTED
     );
 
+    // TODO HTBHF-2682 Rename this method (and others) to createClaim.
     public ClaimResult createOrUpdateClaim(ClaimRequest claimRequest) {
         try {
             EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(claimRequest.getClaimant());
-            if (decision.claimExistsAndIsEligible()) {
-                Claim claim = claimRepository.findClaim(decision.getExistingClaimId());
-                List<UpdatableClaimantField> updatedFields = updateClaim(claim, claimRequest.getClaimant());
-                sendAdditionalPaymentMessageIfNewDueDateProvided(claim, updatedFields);
-                VoucherEntitlement weeklyEntitlement = decision.getVoucherEntitlement().getFirstVoucherEntitlementForCycle();
-                claimMessageSender.sendReportClaimMessageWithUpdatedClaimantFields(claim, decision.getDateOfBirthOfChildren(), updatedFields);
-                return ClaimResult.withEntitlementAndUpdatedFields(claim, weeklyEntitlement, updatedFields, decision.getIdentityAndEligibilityResponse());
+            if (decision.getEligibilityStatus() == EligibilityStatus.DUPLICATE) {
+                Claim claim = createAndSaveClaim(claimRequest, decision);
+                claimMessageSender.sendReportClaimMessage(claim, emptyList(), REJECTED);
+                return ClaimResult.withNoEntitlement(claim);
             }
 
             Claim claim = createAndSaveClaim(claimRequest, decision);
             if (claim.getClaimStatus() == ClaimStatus.NEW) {
                 claimMessageSender.sendInstantSuccessEmailMessage(claim, decision);
                 claimMessageSender.sendNewCardMessage(claim, decision);
                 VoucherEntitlement weeklyEntitlement = decision.getVoucherEntitlement().getFirstVoucherEntitlementForCycle();
-                claimMessageSender.sendReportClaimMessage(claim, decision.getDateOfBirthOfChildren(), ClaimAction.NEW);
+                claimMessageSender.sendReportClaimMessage(claim, decision.getDateOfBirthOfChildren(), NEW);
```

Comment:
Again here, please revert this as its much clearer without the static import of the enum values.

Reply:
Done.